api-spec/protobuf/ark/v1/service.proto

```diff
@@ -123,17 +123,15 @@ message GetInfoResponse {
   int64 round_interval = 4;
   string network = 5;
   int64 dust = 6;
-  string boarding_descriptor_template = 7;
-  repeated string vtxo_descriptor_templates = 8;
-  string forfeit_address = 9;
+  string forfeit_address = 7;
 }
 
 message GetBoardingAddressRequest {
   string pubkey = 1;
 }
 message GetBoardingAddressResponse {
   string address = 1;
-  string descriptor = 2;
+  repeated string tapscripts = 2;
```

Comment:
I would rather add a filed so we retain possibility in the future to also return descriptors

Reply:
indeed